dto/scenario_testrun.go

```diff
@@ -0,0 +1,37 @@
+package dto
+
+import (
+	"time"
+
+	"github.com/checkmarble/marble-backend/models"
+)
+
+type CreateScenarioTestRunBody struct {
+	ScenarioIterationId string `json:"scenario_iteration_id"`
+	ScenarioId          string `json:"scenario_id"`
+	Period              string `json:"period"`
+}
+
+type ScenarionResp struct {
```

Comment:
I'm pretty sure we're going to a need a more complete DTO for test runs (at the latest when we want to display them in the app with their details), so I'd say we create a complete dto mapping for them in this PR already (and return it in the api handler)